mcli/mcli-1b-eval.yaml

```diff
@@ -21,46 +21,39 @@ compute:
 # The below is injected as a YAML file: /mnt/config/parameters.yaml
 parameters:
   run_name: # If left blank, will be read from top YAML name for W&B logging and checkpointing
-
   seed: 1
-  max_seq_len: 2048
-
-  tokenizer:
-    name: EleutherAI/gpt-neox-20b
-    kwargs:
-      model_max_length: ${max_seq_len}
-
-  model:
-    name: mpt_causal_lm
-    init_device: meta
-    d_model: 2048
-    n_heads: 16 # Modified 24->16 so that d_head == 128 to statisfy FlashAttention
-    n_layers: 24
-    expansion_ratio: 4
-    max_seq_len: ${max_seq_len}
-    vocab_size: 50368
-    attn_config:
-      attn_impl: triton
-
-  load_path: # Add your (optional) Composer checkpoint path here!
+  max_seq_len: 1024
```

Comment:
does this yaml run using foundry release 0.2.0. I'm assuming not. We can't change yamls that are pinned to a foundry release out of sync with new releases.